pkg/services/stats/models.go

```diff
@@ -49,32 +49,24 @@ type DataSourceStats struct {
 	Type  string
 }
 
-type GetSystemStatsQuery struct {
-	Result *SystemStats
-}
+type GetSystemStatsQuery struct{}
```

Comment:
Could we potentially remove some of the structs that were updated in this file and refactor the signature of the functions where they are passed in as arguments? Asking since they don't have any fields anymore.

Reply:
As insane as it sounds, I'd say - no.

Some time ago we all agreed that services should always take two parameters in their methods - a context, and a Query/Command. The latter is a struct with all the request parameters to that service method. The fact that we pack all parameters into a struct is partially a legacy reason (we has a message bus before), but partially encourages backwards compatibility - if one adds a new parameter/option to the command -- all the remaining code keeps working.

Thus, empty structs as commands/queries are just fine, they express the intent quite clearly.